wazuh/wazuh_managers/wazuh-worker-sts.yaml

```diff
@@ -39,7 +39,7 @@ spec:
             name: wazuh-conf
         - name: filebeat-certs
           secret:
-            secretName: indexer-certs
+            secretName: fliebeat-tls
```

Comment:
Hello, I tried these changes, and I spotted this typo for the secret name here. 

```
  Normal   Scheduled    31s                default-scheduler  Successfully assigned wazuh/wazuh-manager-worker-0 to node1
  Warning  FailedMount  15s (x6 over 30s)  kubelet            MountVolume.SetUp failed for volume "filebeat-certs" : secret "fliebeat-tls" not found
```

Should be `filebeat-tls`

Thank you

Reply:
@nicolasjulian Thank you very much for the comments. I will check and fix.

Reply:
Hello,

Everything looks good on my end. I've but I corrected a typo locally. However, I noticed that for the current stable image version 4.7.3, the configuration mismatch is (( this is before your latest commit )):


```yaml
uiSettings.overrides.defaultRoute: /app/wazuh
```

Is there has been a change to the default route in newer versions? in 4.9.0?

Reply:
@nicolasjulian could you test once more please? If the path should be changed, I can do it.